app/views/index.html

```diff
@@ -79,6 +79,9 @@ <h2 class="govuk-heading-m">
       }) }}
     </form>
 
+    <p>
+      <a href="/online-booking/choose-date-calendar">Date picker - calendar view</a>
```

Comment:
I've put a temporary link to this page on the index page to make it easier to find - but feel free to remove it!